Emby.Server.Implementations/IO/ManagedFileSystem.cs

```diff
@@ -580,7 +580,11 @@ public virtual List<FileSystemMetadata> GetDrives()
         {
             // check for ready state to avoid waiting for drives to timeout
             // some drives on linux have no actual size or are used for other purposes
-            return DriveInfo.GetDrives().Where(d => d.IsReady && d.TotalSize != 0 && d.DriveType != DriveType.Ram)
+            return DriveInfo.GetDrives()
+                .Where(
+                    d => (d.DriveType == DriveType.Fixed || d.DriveType == DriveType.Network || d.DriveType == DriveType.Removable)
```

Comment:
Which types are causing issues?

Reply:
`Unknown`

Reply:
Shouldn't the check then be `d.DriveType != DriveType.Unknown`? You're filtering out a bunch of types that should work (admittedly RAMDisk is stupid, but that's neither here nor there).

Reply:
We already filtered out ram